src/htable.cpp

```diff
@@ -924,24 +925,24 @@ HeadedTable::HeadedTable(QWidget *parent, int opts) : QWidget(parent)
     // synchronize horizontal scrolling of head and body
     connect(body->horizontalScrollBar(), SIGNAL(valueChanged(int)), head,
             SLOT(scrollSideways(int)));
-    fontChange(font()); // *** need for init
+    fontChange(); // *** need for init
                         //	vp=new VPointer((QWidget *)QObject::parent());
                         //	vp->hide();		//memory leak!!
 }
 
 HeadedTable::~HeadedTable() {}
 
 // ok : move to Qttableview
-void HeadedTable::fontChange(const QFont &oldFont)
+void HeadedTable::fontChange()
 {
     // DEBUG("fontChange()\n");
-    int fontHeight = fontMetrics().height() + 1;
+    int fontHeight = QApplication::fontMetrics().height() + 1;
```

Comment:
Preferences window changes font directly in QApplication while font in widget left unchanged.
See https://github.com/QtDesktop/qps/blob/master/src/prefs.cpp#L365